data/campaigns/Two_Brothers/scenarios/03_Guarded_Castle.cfg

```diff
@@ -693,17 +693,27 @@ Yet for some reason I fear these brothers more. If Mordak were here it would be
 
     [event]
         name=moveto
-
```

Comment:
The scrolling aside, I don't really see what the purpose of this change is. The worst criticism about the original seems to be that after the event, one could say that the unit couldn't have seen Baran inside the cell because their vision only reached up to where the gate is, but I have a hard time seeing that as a real issue (and considering the cell is cave, that could still hold true with this change). This fix seems exactly like the kind of thing which eventually someone will just replace with an equivalent of the original again because it's much simpler that way.